dell/xps/13-9320/webcam.nix

```diff
@@ -0,0 +1,32 @@
+{ config, pkgs, ... }:
+
+{
+  hardware.firmware = let
+    ivsc-firmware = with pkgs; stdenv.mkDerivation rec {
```

Comment:
We might potentially want to move this to nixpkgs if it is also useful for other hardware.

Reply:
ohh, it is in nixpkgs nowadays as well I see.

Reply:
Right. In that case, we should pull it from there.

Reply:
mr created to cover add the necessary symbolic links https://github.com/NixOS/nixpkgs/pull/295508